media/js/firefox/sync.js

```diff
@@ -32,15 +33,15 @@
     };
 
     // Variations 1-5 are Firefox
-    if (window.isFirefox()) {
+    if (client.isFirefox) {
```

Comment:
Actually no need, the new `if (client.isFirefoxDesktop)` below should be enough :+1: